packages/epo-widget-lib/src/widgets/ColorTool/ImageComposite/ImageComposite.tsx

```diff
@@ -0,0 +1,44 @@
+import { FunctionComponent } from "react";
+import { ImageFilter } from "../ColorTool";
+import * as Styled from "../styles";
+
+interface ImageCompositeProps {
+  filters: ImageFilter[];
+}
+
+const ImageComposite: FunctionComponent<ImageCompositeProps> = ({
+  filters,
+}) => {
+  const isFilterActive = (filters: ImageFilter[]) =>
+    filters.some((f) => f.active);
+
+  return (
+    <Styled.ImageContainer>
+      <Styled.BackgroundImage
+        alt=""
```

Comment:
this image is just a background and rarely ever visible to users so setting the alt to `""` will remove it from screenreader visibility